certified-connectors/Signi/apiProperties.json

```diff
@@ -88,7 +88,7 @@
           "description": "Grant type",
           "constraints": {
             "required": "false",
-            "hidden": true,
+            "hidden": "true",

```

Comment:
Please check apiproperties validation errors and change the value of hidden parameter duly.